Gemfile

```diff
@@ -6,8 +6,9 @@ gem 'rails', '~> 5.2.4', '>= 5.2.4.2'
 gem 'pg', '>= 0.18', '< 2.0'
```

Comment:
removed active model serializer and added jbuilder so i could customize my json responses for revenue endpoints